### PR TITLE
Name Metatiles

### PR DESCRIPTION
**Pull request by garakmon**

### CODE REVIEW

File: src/braille_puzzles.c [hunk 1]
```diff
@@ -131,12 +132,12 @@ void UseFlyAncientTomb_Callback(void)
...
-    MapGridSetMetatileIdAt(14, 26, 554);
-    MapGridSetMetatileIdAt(15, 26, 555);
-    MapGridSetMetatileIdAt(16, 26, 556);
-    MapGridSetMetatileIdAt(14, 27, 3634);
-    MapGridSetMetatileIdAt(15, 27, 563);
-    MapGridSetMetatileIdAt(16, 27, 3636);
+    MapGridSetMetatileIdAt(14, 26, METATILE_ID(Cave, SealedChamberEntrance_TopLeft));
+    MapGridSetMetatileIdAt(15, 26, METATILE_ID(Cave, SealedChamberEntrance_TopMid));
+    MapGridSetMetatileIdAt(16, 26, METATILE_ID(Cave, SealedChamberEntrance_TopRight));
+    MapGridSetMetatileIdAt(14, 27, METATILE_ID(Cave, SealedChamberEntrance_BottomLeft) | METATILE_COLLISION_MASK);
```
Comment (PikalaxALT) [inline src/braille_puzzles.c]: Probably not the best to use the `_MASK` constant here

Comment (garakmon) [reply src/braille_puzzles.c]: why not?

Comment (PikalaxALT) [reply src/braille_puzzles.c]: Ironically, it masks what the flags actually represent.

Comment (garakmon) [reply src/braille_puzzles.c]: so is the solution to replace that constant with the raw value?
and is this a problem everywhere or just here?

Comment (PikalaxALT) [reply src/braille_puzzles.c]: Create new defines for each collision direction, and a new define that `or`s the 4 directions. Use the new define shifted by `METATILE_COLLISION_MASK`.

**Comment by garakmon** [comment]
okay I updated this but travis had a meltdown for a minute. should be OK if you restart the build tho

**Pull request merged**